File: gamelogic/src/main/java/com/alta_v2/game/gamelogic/data/npc/NpcModel.java
package com.alta_v2.game.gamelogic.data.npc;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Builder
public class NpcModel {

    private final String id;
    private final float localX;
    private final float localY;
    private final int repeatMovementInterval = 5000;

    @Setter
    private long lastMovementMills;

    @Setter
    private boolean isMovementRunning;

}

File: gamelogic/src/main/kotlin/com/alta_v2/game/gamelogic/data/npc/NpcModel.kt
package com.alta_v2.game.gamelogic.data.npc

data class NpcModel(val id: Int,
                    val localX: Float,
                    val localY: Float,
                    val repeatMovementInterval: Int,
                    val interactionGroupId: Int?) {

    var lastMovementMills: Long = 0
    var isMovementRunning: Boolean = false
}